infoTree/getURLfile.js

```diff
@@ -1,9 +1,14 @@
 import leafURLS from './URLs.js';
 import whereDoesItGo from './whereDoesItGo.js'
 
+const errorPage = 223;
```

Comment:
uh, okay. Not sure why we have a magic number. Returning an error message sound be better, even if this is only temporary. Temporary code has a way of becoming permanent.

Reply:
we only have last json file is URL222, so i was going to reserve URL223 for our error page. This should only occur if the ML model returns a label that doesn't point to any JSON files. This could also be used if the probability returned from the ML model is too low. This should be handled in a different way, but I was using this while testing, since before it was returning undefined when a label wasn't found and crashing the app

Reply:
oh I see. Didn't read far enough to see that this function returns the index. For a numeric error value, I typically see -1 as you wouldn't ever get that as an index.